node/manager.go

```diff
@@ -201,6 +217,6 @@ func (dm *DelegateManager) isDelegate() bool {
 	return false
 }
 
-func hashNodeInfo(msg *iotextypes.ResponseNodeInfoMessage) hash.Hash256 {
-	return hash.Hash256b(byteutil.Must(proto.Marshal(msg.Info)))
+func hashNodeInfo(msg *iotextypes.NodeInfo) hash.Hash256 {
```

Comment:
this was renamed to `NodeInfoCore` in haixiang's latest change in `iotex-proto`, wait for the new release (after another PR he's preparing now)